content/news/035/index.md

```diff
@@ -164,6 +164,19 @@ Interested in contributing? Head over to the
 [botnet_example_bot]: https://github.com/JMS55/botnet/blob/master/example_bot/src/lib.rs
 [botnet_ideas]: https://github.com/JMS55/botnet/discussions/categories/ideas
 
+### [Star Machine]
+
+![Screenshot of a level in Star Machine](star_machine.png)
+
+[Star Machine] by [@Seldom_SE] is a puzzle game built in Bevy, where
+you wire together components to escape each level.
+
+The game, although its development is presently inactive, made
```

Comment:
this sentence looks a little weird (sorry for the late review)

Reply:
Which sentence specifically? I'm new to GitHub reviews haha

Reply:
"The game <...> made a video demo <...>"

Reply:
I'll make note to tweak this in the release PR.